### DISCUSSION TITLE
Slovenian Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#399] [Language Leaders]
Discussion to elect Slovenian language leader(s)

**Comment by andrejs2** [comment]
Any info about Slovenian language leader? Would be useful to get some info if I should proceed with translation or just wasting my time?
I have some PRs open but I'm not sure if anyone can engage since SI language leader is not present.

Thanks!

**Reply by synesthesiam** [reply]
Hi @andrejs2, thank you for contributing! I haven't had a lot of time to keep watch on the PR's here until now, so I apologize for the delay.

Would you be interested in being a language leader for Slovenian?

**Reply by andrejs2** [reply]
Thank you @synesthesiam for your reply. I'm sorry; I didn't mean to bother you, but I noticed a previous discussion where the language leader was already approved but not currently present. I tried to contact them a few weeks ago, but I had no luck. I tried to find native Slovenian developers here on GitHub, but I didn't receive any responses.

I would be willing to take on this responsibility in order to advance Slovenian translations, but I must clarify that I'm not a developer or programmer. I do have some experience with Home Assistant (including YAML and related tasks), and this is my first commit on GitHub. I have a strong belief in the open-source community and this project, but I may require occasional assistance. If this is acceptable to all, I can take on this task temporarily until someone with better skills joins. I understand that I may be a one-man band for now.

**Reply by synesthesiam** [reply]
That's fine, as long as your YAML is able to pass the tests and you can merge PR's :slightly_smiling_face: 
I'll get you added as a language leader and let you merge what you have.

**Reply by andrejs2** [reply]
Thanks for your trust, I'll do my best!

**Reply by tetele** [reply]
@andrejs2 given your history of PRs, please be careful not to merge anything that doesn't belong to Slovenian and even for Slovenian, to only merge what has passing tests. If in doubt, drop me a message here or on Discord.

**Reply by andrejs2** [reply]
Thanks. My initial PRs were a bit messy, I admit, but it's been a steep learning curve for me. I won't make any changes without consulting with you guys first. 👍 Since the SL language is essentially empty in the repo, I'll have to complete all the sentences with other necessary files (common, fixtures, responses) and then prepare it for merging; otherwise, I might break something that's already functioning. So, I anticipate the PR is going to be substantial.

**Comment by dolenec** [comment]
So what is status here?

**Comment by andrejs2** [comment]
Momentary, [this ](https://ohf-voice.github.io/intents/)is the status for Slovenian language. 
Most of intents are translated - > https://github.com/OHF-Voice/intents/tree/main/sentences/sl

**Reply by dolenec** [reply]
Great.. if there is need for some help notify me..
